src/main/kotlin/org/rust/lang/core/resolve/RustResolveEngine.kt

```diff
@@ -280,8 +281,11 @@ private class Resolver {
 
         override var matched: RustNamedElement? = null
 
-        override fun visitModItem(o: RustModItem) {
-            seek(o.itemList)
+        override fun visitModItem(mod: RustModItem) {
+            seek(mod.itemList)
+            if (matched == null) {
+                seekInjectedItems(mod)
```

Comment:
Why do we seek injected items inside of every module, while `extern crate` is implicitly injected in the root one?